server/httpin/admin.go

```diff
@@ -15,17 +15,22 @@ import (
 // manifest store, making the App installable. The resulting listed manifest
 // will combine the deployment information from the prior listing, and the new
 // manifests as follows:
-//   1. The "core" manifest (except Deploy) is updated to the new values.
-//   2. Deploy types from the previously listed manifest are updated from the new manifest, or preserved.
-//   3. Deploy types specified in "add_deploys" are copied from the new manifest.
-//   4. "remove"
-//   Path: /api/v1/add-listed-app
-//   Method: POST
-//   Input: JSON{
-//      Manifest...
-//      "add_deploys": []string e.g. ["aws_lambda","http"]
-//      "remove_deploys": []string e.g. ["aws_lambda","http"]
-//   Output: The updated listing manifest
+//
+//  1. The "core" manifest (except Deploy) is updated to the new values.
+//  2. Deploy types from the previously listed manifest are updated from the new manifest, or preserved.
+//  3. Deploy types specified in "add_deploys" are copied from the new manifest.
+//  4. "remove"
+//
+// _
```

Comment:
What does this do?

Reply:
It separates the code block from the numbered list. Without the character the two blocks get combined, and the code block becomes part of the 4. item.